src/notes/notes_controller.ts

```diff
@@ -0,0 +1,22 @@
+import { getAllPredefinedNotes } from "./get_all_notes";
+import {Request, Response} from 'express';
+import { getPredifinedUsers } from "./get_all_users";
+
+export class NotesController{
+    
+      static getAllNotes(request : Request, response : Response){
+        let allNotes = getAllPredefinedNotes();
```

Comment:
Since we are not modifying all notes, then it should be const instead of let e.g.
`const allNotes = getALlPredefinedNotes()`